Tests/Common/Background Queue/QueueTest.swift

```diff
@@ -16,16 +16,9 @@ class QueueTest: UnitTest {
         super.setUp()
 
         sdkConfigStoreMock.config = SdkConfig()
-
-        queue = CioQueue(
-            siteId: testSiteId,
-            storage: storageMock,
-            runRequest: runRequestMock,
-            jsonAdapter: jsonAdapter,
-            logger: log,
-            sdkConfigStore: sdkConfigStoreMock,
-            queueTimer: queueTimerMock
-        )
+        queue = CioQueue(siteId: testSiteId, storage: storageMock, runRequest: runRequestMock, jsonAdapter: jsonAdapter,
```

Comment:
minor style note prefer the old way of param per line

Reply:
Updated. This was a result of the merge. Thanks for pointing out.